src/main/java/messenger/MessengerNotification.java

```diff
@@ -7,18 +7,30 @@
 
 public class MessengerNotification implements Notification {
 
+    private String senderId;
```

Comment:
what are these are/planned to be used for?

Reply:
HttpRequest consists only sender and receiver ids (more privacy etc). Our server recognizes them and adds corresponding names to proper Notification.
It is mainly used for showing what can be done during processing notification.